src/commands/scaling/addScaleRule/addScaleRule.ts

```diff
@@ -0,0 +1,23 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { IActionContext } from "@microsoft/vscode-azext-utils";
+import { ext } from "../../../extensionVariables";
+
+import { ScaleRuleGroupTreeItem } from "../../../tree/ScaleRuleGroupTreeItem";
+import { ScaleRuleTreeItem } from "../../../tree/ScaleRuleTreeItem";
+
+
+export async function addScaleRule(context: IActionContext, node?: ScaleRuleGroupTreeItem): Promise<void> {
+    if (!node) {
+        node = await ext.tree.showTreeItemPicker<ScaleRuleGroupTreeItem>(new RegExp(ScaleRuleGroupTreeItem.contextValue), context);
+    }
+
+    const srNode: ScaleRuleTreeItem = await node.createChild(context);
+    // void showContainerAppCreated(srNode);
+    console.log(srNode);
```

Comment:
Should these be removed?

Reply:
Yup, I that was from an unfinished WIP-commit :)